qa/src/test/resources/java-web-app-and-db-with-function.yaml

```diff
@@ -17,18 +17,20 @@
 # under the License.
 #
 name: java-cluster-db-example
-location: localhost
 services:
-- type: org.apache.brooklyn.entity.webapp.ControlledDynamicWebAppCluster
+- serviceType: org.apache.brooklyn.entity.webapp.ControlledDynamicWebAppCluster
```

Comment:
Why change this? `type` should work as well. This file seems unrelated to the rest of the pull request.